app/controllers/admin_controller.rb

```diff
@@ -40,6 +39,9 @@ def new_instructor
   #     end
   #   end
   # end
+ # def show
```

Comment:
It's not significant.. I put that to try some redirects. It didn't work out so I just commented it. Will remove it now